addons/point_of_sale/tests/test_frontend.py

```diff
@@ -1109,6 +1109,24 @@ def test_limited_product_pricelist_loading(self):
         self.main_pos_config.open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'limitedProductPricelistLoading', login="accountman")
 
+    def test_ship_later_minimun_rights(self):
+        minimum_pos_user = self.env['res.users'].create({
+            'name': 'A simple PoS man!',
+            'login': 'minimum_pos_user',
+            'password': 'minimum_pos_user',
+            'groups_id': [
+                (4, self.env.ref('base.group_user').id),
+                (4, self.env.ref('point_of_sale.group_pos_user').id),
+            ],
+        })
```

Comment:
from what i see, the diff between this `minimum_pos_user` and the `pos_user` defined in this file is the fact that the current `pos_user` from this file also has the `account.group_account_invoice` group. With the change that you made in this commit, the `pos_user` shouldn't have to be part of the `account.group_account_invoice` anymore so, in that case, you wouldn't need this `minimum_pos_user`

Reply:
Indeed that makes, sense. I'll remove the tests and just change the pos_user access rights

Reply:
please mention this change in the commit message too. It's quite a substantial difference